metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "conceptlist": "Tryon-1983-324",
-    "citation": "Tryon, D. T. and Hackman, B. D. (1983): Solomon Islands Languages: An internal classification. Canberra: Pacific Linguistics.",
+    "citation": "Tryon, Durrell T. and Hackman, Brian D. (1983): Solomon Islands Languages: An internal classification. Canberra: Pacific Linguistics.",
```

Comment:
There name is misspelled here so the dataset needs a rebuild after the metadata change.